03LessonThree/homework/git-scm.html

```diff
@@ -7,21 +7,21 @@
 </head>
 
 <body>
-    <div id="header">
-        <div id="nav">
-            <span><a href="#">Home</a></span>
-            <span><a href="#">How does it work</a></span>
-            <span><a href="#">Contact Us</a></span>
-        </div>
-    </div>
-    <div id="main">
-        <div id="section">
+    <header>
+        <main>
+            <article><a href="#">Home</a></article>
+            <article><a href="#">How does it work</a></article>
+            <article><a href="#">Contact Us</a></article>
+        </main>
+    </header>
+    <main>
+        <header>

```

Comment:
i think here it wasn't a header but instead another element. look well to the value inside the attribute id="..." and it will tell you what element you should have it replace it with.